libs/components/ag-grid/src/lib/modules/ag-grid/ag-grid-data-manager-adapter.directive.ts

```diff
@@ -418,30 +419,43 @@ export class SkyAgGridDataManagerAdapterDirective
   }
 
   #applyColumnWidths(): void {
+    if (!this.agGridList?.first || this.agGridList.first.api.isDestroyed()) {
+      return;
+    }
```

Comment:
There was a test warning b/c the grid was destroyed when this was firing. This change fixes that.